watsononlinestore/watson_online_store.py

```diff
@@ -38,6 +38,10 @@ def __init__(self, slack_client, channel):
         self.channel = channel
 
     def send_message(self, message):
+        """ Sends message via Slack API
+
+        param: string message The message to be sent to slack
```

Comment:
:param message: <desc>
or
:param str message: <desc>
but definitetly colon before param and after message